kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -1,15 +1,223 @@
 <template>
 
-  <video autoplay=true src='http://vid297.photobucket.com/albums/mm238/daystar170/RickRoll.mp4' controls=true></video>
+  <div>
+    <div class="videowrapper" v-el:videowrapper>
+      <video v-el:video class="video-js vjs-default-skin" >
+        <source :src="videoUrl" type='video/mp4'>
+        <track kind="captions" :src="captionsUrl" :srclang="captionsLang" :label="captionsLabel">
+      </video>
+    </div>
+  </div>
 
 </template>
 
 
 <script>
 
-  module.exports = {};
+  const videojs = require('video.js');
+  require('./replayPlayForward');
+  let videoPlayer = null;
+  module.exports = {
+
+    ready() {
+      videoPlayer = videojs(this.$els.video, {
```

Comment:
(It would probably be best to explicitly test adding multiple video renderers on a single page with independent videos states, and make sure things still work.)